src/cron/schedule-messages.js

```diff
@@ -36,20 +63,22 @@ const scheduleMessage = ({ client, channel, message, datetime, timeZone }) => {
 };
 
 /**
- * Schedules multiple messages to be sent to a Discord channel based on the provided message times.
+ * Schedules multiple messages to be sent to a Discord channel based on the provided schedule.
  *
  * @param {Client} client - The Discord.js client instance.
- * @param {Array<{channel: string, datetime: string, messsage: string}>} scheduledMessages - Array of objects containing cron time and message message.
+ * @param {Array<{message: string, variables: {channel: string, time: string, timezone: string, days?: string}}>} scheduledMessages -
  */
-const scheduleMessages = (client, scheduledMessages) => {
-  scheduledMessages.forEach((scheduledMessage) => {
-    const { channel, datetime, message } = scheduledMessage;
+const scheduleMessages = ({ client, messages }) => {
+  messages.forEach((msg) => {
+    const { message, variables } = msg;
+    const { channel, cronExpression, timezone } =
+      convertToCronExpression(variables);
     scheduleMessage({
       client,
-      channel,
-      message,
-      datetime,
-      timeZone: SCHEDULE_MESSAGES.timeZone,
+      channel: channel,
+      message: message,
```

Comment:
These 2 are not needed to be explicit
```suggestion
      channel,
      message,
```